index.js

```diff
@@ -40,11 +42,11 @@ function getNodeModuleDirectory(directory) {
 }
 
 module.exports = (options = {}) => {
-	if (process.env.CACHE_DIR) {
-		return useDirectory(path.join(process.env.CACHE_DIR, 'find-cache-dir'), options);
+	if (env.CACHE_DIR && !/^true|false|1|0$/.test(env.CACHE_DIR)) {
```

Comment:
This regex is incorrect. It will match, for example, `truefalse`. It's also an unnecessary use of regex. Just use an array and Array#includes.

Reply:
Fixed!